drivers/volume/portworx/portworx.go

```diff
@@ -1314,17 +1321,18 @@ func (d *portworx) setDriver() error {
 	var err error
 	var endpoint string
 
-	// Try portworx-service first
-	endpoint, err = d.schedOps.GetServiceEndpoint()
-	if err == nil && endpoint != "" {
-		if err = d.testAndSetEndpoint(endpoint); err == nil {
-			d.refreshEndpoint = false
-			return nil
-		}
-		logrus.Infof("testAndSetEndpoint failed for %v: %v", endpoint, err)
-	} else if err != nil && len(node.GetWorkerNodes()) == 0 {
-		return err
-	}
+	/*
```

Comment:
why we have commented code here ?

Reply:
good catch. fixed. had commented for dev test to save time.